tests/tests.py

```diff
@@ -42,467 +46,447 @@ def _reset_quotas(base_dir, projects_file, projid_file, homedirs):
                 os.remove(os.path.join(base_dir, d, f))
         os.rmdir(os.path.join(base_dir, d))
 
+
+@pytest.fixture
+def quota_manager(tmp_path):
+    quota_manager = QuotaManager.instance(
+        projid_file=os.fspath(tmp_path / "projid"),
+        projects_file=os.fspath(tmp_path / "projects"),
+        min_projid=1000,
+        hard_quota=1000 / 1024**2,
```

Comment:
This is ugly (a non-integral float value), but it preserves the previous 1000 byte value.